plugins/woocommerce/includes/class-wc-tracker.php

```diff
@@ -530,14 +588,47 @@ private static function get_orders_by_gateway() {
 		);
 
 		$orders_by_gateway_currency = array();
+
+		$orders_by_gateway = self::group_objects_by_key(
```

Comment:
I think we should give some context why this logic exists:

```suggestion
		// Group data by payment method. Strip any transaction identifiers from method descriptions.
		$orders_by_gateway = self::group_objects_by_key(
```

Reply:
Sure will give a detailed explanation.